utils/config/config.go

```diff
@@ -747,6 +748,12 @@ func (serverDetails *ServerDetails) CreateAccessAuthConfig() (auth.ServiceDetail
 	return serverDetails.createAuthConfig(pAuth)
 }
 
+func (serverDetails *ServerDetails) CreateJfConnectAuthConfig() (auth.ServiceDetails, error) {
+	pAuth := accessAuth.NewAccessDetails()
+	pAuth.SetUrl(utils.AddTrailingSlashIfNeeded(serverDetails.Url) + "jfconnect/")
```

Comment:
woudln't this cause double `jfconnect/` on the URL?
because here you create the base URL `ecosys.jfrog.io/access/jfconnect` and on the PostMetric you add `jfconnect/api/v1/backoffice/metrics/log`

Reply:
I think you're right @sverdlov93. Good catch 👍